wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceRuntimes.java

```diff
@@ -449,6 +452,9 @@ public void run() {
   @Traced
   public CompletableFuture<Void> stopAsync(Workspace workspace, Map<String, String> options)
       throws NotFoundException, ConflictException {
+    TracingTags.WORKSPACE_ID.set(workspace.getId());
+    TracingTags.USER_ID.set(() -> EnvironmentContext.getCurrent().getSubject().getUserId());
```

Comment:
If a workspace is stopped by `ActivitManager` or `ConsistencyChecker` then `"0000-00-0000"` will appear here.
Please take a look how stopped by are evaluated https://github.com/eclipse/che/blob/6553ddfbac6e78b8f3857158668f99ac84c2ea2e/wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceRuntimes.java#L470-L472
Maybe it would be more helpful to see cron job here instead of `"0000-00-0000"`

Reply:
reworked